test/morgan.js

```diff
@@ -1067,7 +1219,7 @@ describe('morgan()', function () {
           var cb = after(2, function (err, res, line) {
             if (err) return done(err)
             var masked = line.replace(/\x1b\[(\d+)m/g, '_color_$1_')
-            assert.equal(masked, '_color_0_GET / _color_0_- _color_0_- ms - -_color_0_')
```

Comment:
Hmm. Well, the color resets is a necessary feature we must have in the `dev` token, so adding that back somehow isn't even a question in my mind.

Adding the color reset in the other tokens absolutely cannot happen, either, as all those other formats are being written out to log files and the like and this will be very heavily breaking, since the resets are actual characters that are going to end up in all those log files all of a sudden.

I'm not sure what a good solution is right now, because I'd have to think about it. If you can proposal any additional ideas, I'd be happy to read them!